ansible/group_vars/all

```diff
@@ -106,7 +106,7 @@ nginx_vhosts:
         fastcgi_param PATH_INFO $path_info;
         fastcgi_param PATH_TRANSLATED $document_root$path_info;
         include snippets/fastcgi-php.conf;
-        fastcgi_pass unix:/var/run/php5-fpm.sock;
+        fastcgi_pass {{ fastcgi_pass_sock }};
```

Comment:
I didn’t know the socket was referenced elsewhere. Knowing now that the php config files reference it, I think we should leave it as is i.e. override a single variable in the vhosts config that we’re already generating entirely within ansible.